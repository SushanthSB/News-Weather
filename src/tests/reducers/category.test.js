import Category from '../../reducers/category';

it("Data shouldn't change",() => {
	expect(Category()).toMatchSnapshot();
})