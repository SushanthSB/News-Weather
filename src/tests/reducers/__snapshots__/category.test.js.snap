// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data shouldn't change 1`] = `
Array [
  Object {
    "id": 1,
    "name": "General",
  },
  Object {
    "id": 2,
    "name": "Politics",
  },
  Object {
    "id": 3,
    "name": "Business",
  },
  Object {
    "id": 4,
    "name": "Sports",
  },
  Object {
    "id": 5,
    "name": "Technology",
  },
  Object {
    "id": 6,
    "name": "Entertainment",
  },
  Object {
    "id": 7,
    "name": "Music",
  },
  Object {
    "id": 8,
    "name": "Gaming",
  },
]
`;
